### PR TITLE
add official register numbers etc, add kajaani, fix one typo

### PR DESCRIPTION
**Pull request by tswfi**

### CODE REVIEW

File: _data/kaupungit.yml [hunk 1]
```diff
@@ -11,11 +11,17 @@
   lat: 60.22199
   lon: 24.85194
   osoite: Takkatie 18, takaovi
+  officialname: Helsinki Hacklab ry
+  ytunnus: 2346395-6
+  regno: 203.110
```
Comment (olmari) [inline _data/kaupungit.yml]:
```suggestion
  regno: 203.110
```

I'm not sure register numbers are that relevnat info anymore as everything is officially Y-tunnus now.

File: _data/kaupungit.yml [hunk 2]
```diff
@@ -96,6 +120,12 @@
   lat: 65.02556
   lon: 25.48531
   osoite: Valtatie 67, takaovi
+  officialname: Tarlab Oulu ry
+  ytunnus: 3071183-1
+  regno: 207.088
+  registeredin: Turku
```
Comment (olmari) [inline _data/kaupungit.yml]: Oulu

**Review by olmari — CHANGES_REQUESTED** [review]
Other than these, didn't see anything else popping at a glance.

File: _data/kaupungit.yml [hunk 3]
```diff
@@ -263,3 +353,23 @@
   logo: "seinajoki.png"
   irc: "#SjkHacklab"
   irc_network: "Freenode"
+
+- kaupunki: Kajaani Hacklab
```
Comment (olmari) [inline _data/kaupungit.yml]: General consensus is that Kajaani doesn't mainly either exist or doesn't want to collaborate with us, so remobe whole Kajaani. In the chance they sometimes answer to us then we'll think again.

**Review by olmari — CHANGES_REQUESTED** [review]
Remove Kajaani for now

File: _data/kaupungit.yml (hunk 3 above)
Comment (tswfi) [reply _data/kaupungit.yml]: done in 9973a83

File: _data/kaupungit.yml (hunk 2 above)
Comment (tswfi) [reply _data/kaupungit.yml]: done in 9973a83

File: _data/kaupungit.yml (hunk 1 above)
Comment (tswfi) [reply _data/kaupungit.yml]: yeah, we can probably leave them out. I have never needed them for anything ever :)

Comment (tswfi) [reply _data/kaupungit.yml]: and done in 9f9f3dd

**Review by olmari — APPROVED** [review]
LGTM

**Pull request merged**